.gitignore

```diff
@@ -58,3 +58,6 @@ data/
 # Local environment overrides
 .env
 settings.ini
+
+# l10n
+locale/
```

Comment:
For now, it should get ignored.

When you set browsercompat up on Pontoon, you'll need to figure out whether you're going to do a separate git repository for strings or have the strings in this repo. If you have them in this repo, then you'll remove this line then.